packages/Alert/src/alert.scss

```diff
@@ -112,7 +113,6 @@
         /* Styles */
         &--card {
             background-color: var(--ids-alert-background-color-light);
-            border-radius: var(--ids-alert-border-radius);
```

Comment:
does moving this do unwanted things to the old style alert?

Reply:
The old should have border radius according to KC, he went back in the Figma as fas as possible and it was an issue.